website/docs/cli.md

```diff
@@ -5,10 +5,22 @@ last_modified_at: 2021-08-18T15:59:57-04:00
 ---
 
 ### Local set up
-Once hudi has been built, the shell can be fired by via  `cd hudi-cli && ./hudi-cli.sh`. A hudi table resides on DFS, in a location referred to as the `basePath` and
+Once hudi has been built, the shell can be fired by via  `cd hudi-cli && ./hudi-cli.sh`.
+
+Optionally in release `0.13.0` we have now added another way of launching the `hudi cli`, which is using the `hudi-cli-bundle`.
```

Comment:
Can we make `hudi-cli-bundle` the default way to use Hudi CLI for Spark 3?  Make the instruction for `hudi-cli-bundle` a separate section. Also note down that for Spark 2 the user need to follow the other instruction of building `hudi-cli` module.